airbyte-integrations/connectors/source-file/pyproject.toml

```diff
@@ -21,7 +21,7 @@ html5lib = "==1.1"
 beautifulsoup4 = "==4.11.1"
 openpyxl = "==3.0.10"
 google-cloud-storage = "==2.5.0"
-pandas = "==1.4.3"
+pandas = "1.5.0"
```

Comment:
Could we use the [latest version](https://pypi.org/project/pandas/) of `pandas` in this connector?

Reply:
updated pandas to latest version